changes/feature-6815_nicknym_validation_level

```diff
@@ -0,0 +1 @@
+-  fetched keys from other domain than it's own are set as 'Weak Chain' validation level (Closes: #6815)
```

Comment:
It should be its, because I'm talking about the user provider, maybe I should rephrase it better.